src/app/Scenes/SellWithArtsy/Inquiry/SWAInquiryScreen.tsx

```diff
@@ -0,0 +1,97 @@
+import { useNavigation } from "@react-navigation/native"
+import { FancyModal } from "app/Components/FancyModal/FancyModal"
+import { FancyModalHeader } from "app/Components/FancyModal/FancyModalHeader"
+import { FormikProvider, useFormik } from "formik"
+import { CloseIcon } from "palette"
+import { useEffect, useState } from "react"
+import { ArtsyKeyboardAvoidingViewContext } from "shared/utils"
+import * as Yup from "yup"
+import { InquiryForm } from "./InquiryForm"
+import { InquiryFormAbandonEdit } from "./InquiryFormAbandonEdit"
+import { SWAInquiryConfirmation } from "./SWAInquiryConfirmation"
+
+interface InquiryScreenProps {
+  name: string
+  email: string
+  phone: string
+}
+
+export interface InquiryFormikSchema {
+  name: string
+  email: string
+  phoneNumber: string
+  message: string
+}
+
+const ValidationSchema = Yup.object().shape({
+  name: Yup.string().required("Name field is required").min(1, "Name field is required"),
```

Comment:
```suggestion
  name: Yup.string().trim().required("Name field is required").min(1, "Name field is required"),
```
it's a nice pattern to always trim input fields